### DISCUSSION TITLE
Support badging for SettingsCard control

### DISCUSSION THREAD

**Discussion by jp-weber** [CommunityToolkit/Windows#437] [Ideas]
The SettingsCard control should support badging, to allow for indication of a disabled setting for example a setting which is controlled via MDM.

Mockup:
![grafik](https://github.com/CommunityToolkit/Labs-Windows/assets/10946850/eb11c63a-e630-4db3-bf6a-3122c2377140)

**Comment by Wandtket** [comment]
My organization could definitely make use of this.

**Comment by michael-hawker** [comment]
Moving to the main repo as that's where this control lives now. FYI @niels9001, interesting idea. Didn't NavigationView do something for his not long back or am I mis-remembering? If so, I wonder if we could adopt a similar...

- InfoBadge - https://github.com/microsoft/microsoft-ui-xaml/issues/2963
  - Not sure if this was also supposed to be closed? https://github.com/microsoft/microsoft-ui-xaml/issues/2140
- https://learn.microsoft.com/windows/winui/api/microsoft.ui.xaml.controls.infobadge
- Think it's this property on the NavigationViewItem:
  - https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.navigationviewitem.infobadge

**Comment by niels9001** [comment]
Love this idea. I guess we need to figure out what the best position would be to display the badge. Following what NavigationView is doing maybe?
